public/app/plugins/panel/singlestat/module.ts

```diff
@@ -653,7 +654,7 @@ class SingleStatCtrl extends MetricsPanelCtrl {
           window.location.href = linkInfo.href;
         } else {
           $timeout(() => {
-            $location.url(linkInfo.href);
+            $location.url(locationUtil.stripBaseFromUrl(linkInfo.href));
```

Comment:
This is weird, I know. But somehow $location service has base url set, hence the url has double base urls in it. @torkelo do you know why $location behaves like this?

Reply:
No, cant remember a reason

Reply:
was trying to find some location service base config, but no trace. Maybe this is just how it is 🤷‍♀